pkg/pub_package_reader/lib/pub_package_reader.dart

```diff
@@ -327,6 +328,9 @@ Iterable<ArchiveIssue> forbidPreReleaseSdk(Pubspec pubspec) sync* {
   }
 }
 
+const _pluginDocsUrl =
```

Comment:
Let's move this into `shared/urls.dart`.

Reply:
Would that not make pub_package_reader depend on app?

Reply:
Ah, true, never mind.